@here/harp-vectortile-datasource/lib/adapters/omv/OmvDataAdapter.ts

```diff
@@ -417,17 +430,26 @@ export class OmvDataAdapter implements DataAdapter, OmvVisitor {
                     currentRing.push(command.position);
                 } else if (isClosePathCommand(command)) {
                     if (currentRing !== undefined && currentRing.length > 0) {
+                        const currentRingWinding = Math.sign(ShapeUtils.area(currentRing));
```

Comment:
Could you add a unit test to cover this?

Reply:
done - I created `OmvDataAdapterTest.ts` testing 2 MultiPolygons: one with correct winding and another with opposite winding